routes/v1.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+
+var express = require('express');
+var fs = Promise.promisifyAll(require('fs'));
+var yaml = require('js-yaml');
+
+
+/**
+ * The main router object
+ */
+var router = express.Router();
+
+/**
+ * The main application object reported when this module is require()d
+ */
+var app;
+
+
+/**
+ * GET /conf
+ * Reads and returns the configuration file used to
+ * start the service. IMPORTANT NOTE: This is a sample
+ * only for demostrating promises, you should not
+ * expose this endpoint in real services.
```

Comment:
Definitely. I put it there more as a place holder than the definitive version. Will address that soon (either in this or another PR)